client/my-sites/activity/activity-log/index.jsx

```diff
@@ -357,6 +361,25 @@ class ActivityLog extends Component {
 		);
 	}
 
+	supportsRealtimeBackup() {
```

Comment:
What do you think about moving this to a lib function? This seems like something that could be reused across components.

Reply:
See https://github.com/Automattic/wp-calypso/pull/38473#discussion_r359313708